CodeGeneration/Sources/SyntaxSupport/DeclNodes.swift

```diff
@@ -461,6 +461,7 @@ public let DECL_NODES: [Node] = [
           .keyword(.private),
           .keyword(.public),
           .keyword(.reasync),
+          .keyword(._resultDependsOn),
```

Comment:
Looking at your examples, the decl modifier can only be `_resultDependsOnSelf`, not `_resultDependsOn`, right? So this should be

```suggestion
          .keyword(._resultDependsOnSelf),
```

Reply:
Fixed.